From c019ad5f377a2d2f95a0d5f672421538f995503e Mon Sep 17 00:00:00 2001
From: Alex Deucher <alexander.deucher@amd.com>
Date: Fri, 14 Oct 2016 15:30:17 -0400
Subject: [PATCH 1339/1722] drm/amd/dal: make low_power_tiling_mode a uint32_t
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The code sets multiple values to it, not just true/false.
Fixes a warning:
drivers/gpu/drm/amd/amdgpu/../dal/dc/calcs/bandwidth_calcs.c: In function ‘calculate_bandwidth’:
drivers/gpu/drm/amd/amdgpu/../dal/dc/calcs/bandwidth_calcs.c:569:41: warning: comparison of constant ‘2’ with boolean expression is always false [-Wbool-compare]
   else if (dceip->low_power_tiling_mode == 2) {
                                         ^

Change-Id: I4efa40087a2461affe2e2dcf26623c268de20c69
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
Acked-by: Harry Wentland <harry.wentland@amd.com>
---
 drivers/gpu/drm/amd/dal/dc/inc/bandwidth_calcs.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/dal/dc/inc/bandwidth_calcs.h b/drivers/gpu/drm/amd/dal/dc/inc/bandwidth_calcs.h
index e7fd471..301f3d5 100644
--- a/drivers/gpu/drm/amd/dal/dc/inc/bandwidth_calcs.h
+++ b/drivers/gpu/drm/amd/dal/dc/inc/bandwidth_calcs.h
@@ -130,7 +130,7 @@ struct bw_calcs_dceip {
 	bool dmif_pipe_en_fbc_chunk_tracker;
 	struct bw_fixed dmif_request_buffer_size;
 	uint32_t lines_interleaved_into_lb;
-	bool low_power_tiling_mode;
+	uint32_t low_power_tiling_mode;
 	uint32_t chunk_width;
 	uint32_t number_of_graphics_pipes;
 	uint32_t number_of_underlay_pipes;
-- 
2.7.4

